ccm_web/client/src/App.test.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react'
 import { render, screen } from '@testing-library/react'
-import App from './App'
 
-test('renders learn react link', () => {
-  render(<App />)
-  const linkElement = screen.getByText(/learn react/i)
-  expect(linkElement).toBeInTheDocument()
-})
+// test('renders something', () => {
+//   render(<App />)
+//   const linkElement = screen.getByText(/something/i)
+//   expect(linkElement).toBeInTheDocument()
+// })
```

Comment:
I'm not a huge fan of commented code, but this seems like a decent reminder to add some FE tests. We could also just delete the file and testing dependencies for now.